core/field.js

```diff
@@ -154,6 +154,14 @@ Blockly.Field = function(value, opt_validator, opt_config) {
   opt_validator && this.setValidator(opt_validator);
 };
 
+/**
+ * Construct a Field from a JSON arg object.
+ * @param {!Object} _options A JSON object with options.
+ */
+Blockly.Field.fromJson = function(_options) {
+  throw Error("Field subclass doesn't define fromJson");
```

Comment:
Is there anyway we can tell people which field subclass is causing the problem?

Also style nit: single quotes and period in error msg. (I'm surprised lint doesn't catch the quotes).